src/main/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatter.java

```diff
@@ -1,53 +1,88 @@
 package org.jabref.logic.formatter.bibtexfields;
 
 import java.net.URLDecoder;
+import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import org.jabref.logic.cleanup.Formatter;
 import org.jabref.logic.l10n.Localization;
 
 /**
- * Cleanup URL link
+ * Cleanup URL link.
+ * <p>
+ *     Expose string representations URL links clean up logic.
+ * </p>
  */
 public class CleanupUrlFormatter extends Formatter {
 
-    // This regexp find "url=" or "to=" parameter in full link and get text after them
+    /**This regexp find "url=" or "to=" parameter in full link and get text after them*/
     private static final Pattern PATTERN_URL = Pattern.compile("(?:url|to)=([^&]*)");
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Quoting from the first comment at https://stackoverflow.com/a/13327668/873282

> You should only use @inheritDoc if you intend to add to the original superclass documentation. If you merely want it to be duplicated, Javadoc will do that already, noting that the superclass documentation applies to the subclass's overridden method because the subclass provided no additional documentation.

So, please remove that JavaDoc